Algorithms/Encoders/SoundexEncoder.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using System.Linq;
+
+namespace Algorithms.Encoders
+{
+    /// <summary>
+    /// Class for Soundex encoding strings.
+    /// </summary>
+    public class SoundexEncoder
+    {
+        /// <summary>
+        /// Creates a new instance of the SoundexEncoder.
+        /// </summary>
+        public SoundexEncoder() {}
```

Comment:
Codacy found an issue: [Remove this redundant constructor.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6311443)